extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+| platform       | Platform name in lower-case. Use a specific name such as `sentinel-2a` if part of constellation. MUST NOT duplicate `constellation`. | (1.9) |
+
+#### EO (Electro-Optical)
+
+| Field Name     | Description                                                  | Req.   |
+| -------------- | ------------------------------------------------------------ | ------ |
+| eo:cloud_cover |                                                              | 1.17   |
+| eo:bands       | You may include all bands specified in the assets again as an overview. | (1.10) |
+
+#### Processing
+
+| Field Name          | Description                                                  | Req.        |
+| ------------------- | ------------------------------------------------------------ | ----------- |
+| processing:facility |                                                              | 1.15        |
+| processing:software | String likely needs to be split into software name and version number. | 1.13 / 1.15 |
+| processing:lineage  | Additional processing information and parameters, e.g. a description of the processing chain. | 1.15        |
+
+See also the [notes](#notes) regarding the requirements 1.13 and 1.15 for a better understanding of how to use the fields best.
+
+#### Projection
+
+| Field Name                            | Description                                                  | Req.      |
+| ------------------------------------- | ------------------------------------------------------------ | --------- |
+| proj:epsg / proj:wkt2 / proj:projjson | **REQUIRED.** At least one of the three properties `proj:epsg`, `proj:wkt2` or `proj:projjson` is required. It MUST specify the coordinate reference system (1.5) and MAY specify the map projection (1.6). | 1.5 / 1.6 |
+
+#### View
+
+| Field Name           | Description                                                  | Req. |
+| -------------------- | ------------------------------------------------------------ | ---- |
+| view:off_nadir       | **REQUIRED.** The average off-nadir angle, for per-pixel angles, refer to the asset with the key `off-nadir`. Convert to degree, if required. | \*   |
+| view:incidence_angle | **REQUIRED.** The average incidence angle, for per-pixel angles, refer to the asset with the key `incidence-angle`. Convert to degree, if required. | \*   |
+| view:azimuth         | **REQUIRED.** The average azimuth angle, for per-pixel angles, refer to the asset with the key `azimuth`. Convert to degree, if required. | \*   |
+| view:sun_azimuth     | **REQUIRED.** The average sun azimuth angle, for per-pixel angles, refer to the asset with the key `sun-azimuth`. Convert to degree, if required. | \*   |
+| view:sun_elevation   | **REQUIRED.** The average sub elevation angle, for per-pixel angles, refer to the asset with the key `sun-elevation`. Convert to degree, if required. | \*   |
+
+\* = Requirement 2.8 for Surface Temperature (ST) / 2.11 for Surface Reflectance (SR)
```

Comment:
Does this mean sun elevation and azimuth are required for Surface Temperature?
Sun illumination angles don't factor into temperature

Reply:
Yes, they are. Both specs share the same requirement:

> Provide average solar and sensor viewing azimuth and zenith angles.